std/internal/attributes.d

```diff
@@ -0,0 +1,4 @@
+module std.internal.attributes;
+
+// Internal attributes to be to e.g. annotate unittest blocks
```

Comment:
What does this attribute mean? That it only works with betterC, or that it works with regular D or betterC?